### PR TITLE
Operator timer

### PR DESCRIPTION
**Pull request by bowu**
This commit profiles the execution time of each operator. Different instances of the same operator class have different names in the profile.

### CODE REVIEW

File: libquery/test/CMakeLists.txt [hunk 1]
```diff
@@ -8,6 +8,16 @@ target_link_libraries(filter-op-test galois_query)
 add_test(NAME filter-op-test COMMAND filter-op-test)
 set_tests_properties(filter-op-test PROPERTIES LABELS quick)
 
+add_executable(execution-plan-test ExecutionPlanTest.cpp)
```
Comment (roshandathathri) [inline libquery/test/CMakeLists.txt:11]: Intentional?

File: libquery/src/ConstOp.cpp [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #include "querying/ops/ConstOp.h"
 
+uint32_t galois::ConstOp::num_const_ops_ = 0;
```
[the remark this answers is not in the document]
Comment (bowu) [reply libquery/src/ConstOp.cpp:3]: Can we do this in the declaration? It's not static const.

File: libquery/test/CMakeLists.txt (hunk 1 above)
Comment (bowu) [reply libquery/test/CMakeLists.txt:11]: I'll remove it. Actually, I don't know how it was added.

File: libquery/src/ConstOp.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (roshandathathri) [reply libquery/src/ConstOp.cpp:3]: Yes, it need not be const.

File: libquery/test/CMakeLists.txt (hunk 1 above)
Comment (roshandathathri) [reply libquery/test/CMakeLists.txt:11]: Thought so.

File: libquery/src/ConstOp.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (bowu) [reply libquery/src/ConstOp.cpp:3]: It does't work: "Non-const static data member must be initialized out of line".

[the remark this answers is not in the document]
Comment (roshandathathri) [reply libquery/src/ConstOp.cpp:3]: Oh.. my bad! Sorry.

**Review by roshandathathri — APPROVED** [review]
LGTM

**Pull request merged**